extensions/spack-fairsoft/fairsoft/cmd/fairsoft.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+# Copyright 2020 GSI Helmholtz Centre for Heavy Ion Research GmbH, Darmstadt
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from llnl.util.tty.colify import colify
+import llnl.util.tty as tty
+import spack.extensions.fairsoft as ext
+import sys
+
+description = 'manage FairSoft distros'
+section = 'FairSoft distro'
+level = 'short'
+
+
+def avail(args):
+    """show available FairSoft distros"""
+    distros = sorted(ext.get_envs())
```

Comment:
I assume, `get_envs()` is also "to be implemented"?

Reply:
forgot to add the file, now you can see it